contracts/test/TestPurchase.js

```diff
@@ -32,8 +32,8 @@ const AWAITING_PAYMENT = 0 // Buyer hasn't paid full amount yet
 const SHIPPING_PENDING = 1 // Buyer hasn't paid full amount yet
 const BUYER_PENDING = 2 // Waiting for buyer to confirm receipt
 const SELLER_PENDING = 3 // Waiting for seller to confirm all is good
-const IN_DISPUTE = 4 // We are in a dispute
-const REVIEW_PERIOD = 5 // Time for reviews (only when transaction did not go through)
+// const IN_DISPUTE = 4 // We are in a dispute
+// const REVIEW_PERIOD = 5 // Time for reviews (only when transaction did not go through)
```

Comment:
Reviews ended being parts of 2️⃣and 3️⃣. Should this just be removed completely, or are we planning to bring back a post-dispute review period @DanielVF?

Reply:
Imho we don't have to make that decision now - that's why I just commented them out, so we don't have to deal with it in this PR. (I think it's better to just keep this focused on the linter change.)